packages/material-ui-lab/src/SpeedDial/SpeedDial.d.ts

```diff
@@ -63,6 +63,7 @@ export interface SpeedDialProps
   openIcon?: React.ReactNode;
   /**
    * The component used for the transition.
+   * [Follow our guide](/components/transitions/#transitioncomponent-prop) to learn more about the requirements for this component.
```

Comment:
```suggestion
   * [Follow this guide](/components/transitions/#transitioncomponent-prop) to learn more about the requirements for this component.
```

For all cases. (Appreciate you were just following the diff from the related issue).